### PR TITLE
Lower error level of InvalidTickCount

### PR DESCRIPTION
**Pull request by carllin**
#### Problem
`InvalidTickCount` is a block error that occurs as an expected part of the core protocol when blocks are interrupted, so should not show up as an "Error" in logs

#### Summary of Changes
Lower log severity of `InvalidTickCount` 
Fixes #

### CODE REVIEW

File: core/src/replay_stage.rs [hunk 1]
```diff
@@ -629,11 +629,19 @@ impl ReplayStage {
             // errors related to the slot being purged
             let slot = bank.slot();
             warn!("Fatal replay error in slot: {}, err: {:?}", slot, err);
-            datapoint_error!(
-                "replay-stage-mark_dead_slot",
-                ("error", format!("error: {:?}", err), String),
-                ("slot", slot, i64)
-            );
+            if err.is_severity_error() {
```
Comment (garious) [inline core/src/replay_stage.rs]: Did you mean `is_severe_error`?

File: ledger/src/blockstore_processor.rs [hunk 2]
```diff
@@ -260,6 +260,15 @@ pub enum BlockstoreProcessorError {
     InvalidHardFork(Slot),
 }
 
+impl BlockstoreProcessorError {
+    pub fn is_severity_error(&self) -> bool {
+        match self {
```
Comment (garious) [inline ledger/src/blockstore_processor.rs]: Rust 1.42!
`matches!(self, BlockstoreProcessorError::InvalidBlock(BlockError::InvalidTickCount))`

Comment (carllin) [reply ledger/src/blockstore_processor.rs]: oh awesome, thanks!

File: core/src/replay_stage.rs (hunk 1 above)
Comment (carllin) [reply core/src/replay_stage.rs]: I was trying to communicate the severity of the error: info, trace, error

Comment (garious) [reply core/src/replay_stage.rs]: Doesn't make sense to me, but I won't block you

Comment (mvines) [reply core/src/replay_stage.rs]: I'd rather just remove `is_severity_error()` and inline the `matches!`.  There's no other use for `is_severity_error()` currently, so no need to build a new generic thing for something that is very specific

Comment (carllin) [reply core/src/replay_stage.rs]: sg, removed!

File: core/src/replay_stage.rs [hunk 3]
```diff
@@ -629,11 +630,22 @@ impl ReplayStage {
             // errors related to the slot being purged
             let slot = bank.slot();
             warn!("Fatal replay error in slot: {}, err: {:?}", slot, err);
-            datapoint_error!(
-                "replay-stage-mark_dead_slot",
-                ("error", format!("error: {:?}", err), String),
-                ("slot", slot, i64)
-            );
+            if matches!(
```
Comment (garious) [inline core/src/replay_stage.rs]: Haha, now `if let` will suffice

**Pull request merged**